lib/upipe-freetype/upipe_freetype.c

```diff
@@ -202,6 +202,10 @@ static void upipe_freetype_input(struct upipe *upipe, struct uref *uref, struct
 
     ubuf_pic_clear(ubuf, 0, 0, -1, -1, 0);
 
+    size_t stride_a, stride_y;
+    ubase_assert(ubuf_pic_plane_size(ubuf, "y8", &stride_y, NULL, NULL, NULL));
```

Comment:
While ubase_assert() is OK for unit tests, asserting in a pipe is a bad pratice, and also if the user compiles without NDEBUG your command isn't executed. So you should replace those with UBASE_RETURN or UBASE_FATAL.